files/image_config/rsyslog/rsyslog-config@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Update rsyslog configuration
```

Comment:
i do not understand why when need multi-instance for rsyslog.

Reply:
Removed multiple instances for rsyslog service. Changes required for rsyslog service support for multi-asic platform will be added in a different PR.